app/assets/javascripts/maps.js

```diff
@@ -23,167 +23,71 @@ function initMap(x, y, image, draggable, callback){
   var myLatLng = new google.maps.LatLng(x, y);
 
   var currentLocation = new google.maps.Marker({
-	  position: myLatLng,
-	  draggable: draggable,
-	  map: map,
-	  icon: image
+    position: myLatLng,
+    draggable: draggable,
+    map: map,
+    icon: image
   });
 
   currentLocation.addListener("dragend", callback);
 }
 
-
 function getPoint(string, callback){
-
-
-		//google Geocoding API
-		var url = "http://maps.googleapis.com/maps/api/geocode/json?address=" + string.replace(" ", "+");
-		$.get( url, function( data ) {
-			if(data && data.results){
-				if(data.results.length > 0){
-					callback(data.results[0].geometry.location);
-				}
-			}
-		});
+    // Google Geocoding API
+    var url = "http://maps.googleapis.com/maps/api/geocode/json?address=" + string.replace(" ", "+");
+    $.get( url, function( data ) {
+      if(data && data.results){
+        if(data.results.length > 0){
+          callback(data.results[0].geometry.location);
+        }
+      }
+    });
 }
 
 function placeMarker(lat, lng, content, number){
+  // Create Icon
+  var circleIcon = {
+    url: circleMarkerImage,
+    size: new google.maps.Size(37, 37),
```

Comment:
Where are these values coming from? Can we move them out into constants `DEFAULT_WIDTH` or something?

Reply:
I measured the size of the image. It's in a constant now.